sdk/storage/Azure.Storage.Files.Shares/src/ShareClientConfiguration.cs

```diff
@@ -14,6 +14,8 @@ internal class ShareClientConfiguration : StorageClientConfiguration
 
         public TransferValidationOptions TransferValidation { get; internal set; }
 
+        public ShareAudience Audience { get; internal set; }
```

Comment:
Why does this have to get added to the configuration?

Reply:
The protected static method needs that audience to get saved somewhere in the client so it can access it when making the HttpAuthorization.